src/taco-cli/cli/logo.ts

```diff
@@ -8,7 +8,11 @@
 // NOTE: This file is intended to run on-require, and print the logo, only once.
 // We avoid using logger.log and other package dependencies because this is required in
 // a couple of places including post-install, and we don't want to drag in all the infrastructure
+
+/* tslint:disable:no-var-requires */
+// var require needed to require package json
```

Comment:
An alternative is to just go with JSON.parse(fs.readFileSync()) instead of require(); not sure which I prefer more.